src/main/resources/MessagingDescriptor.json

```diff
@@ -0,0 +1,28 @@
+{
+  "publications": [
+    {
+      "eventType": "ITEM_CHECK_OUT",
```

Comment:
Does it make sense to have `CIRCULATION` prefix for all event names, i.e. `CIRCULATION_ITEM_CHECK_OUT`?

This should guaranty name uniqueness.

Reply:
There's no standard for event names right now, but there's a JIRA ticket to develop them - FOLIO-2408. What we agreed on currently is to use past tense (in the comments https://wiki.folio.org/display/DD/Automated+Patron+Blocks+feature+design) and I see that I forgot to change it here, thanks for highlighting this fragment.
We also thought about using module name as prefix, but we're not sure if it makes sense to connect events to a particular module.

Reply:
Thanks, @alexanderkurash. I'm fine with both naming conventions, assuming that it was agreed.

Reply:
I suggested the past tense naming. There are no agreed standards or conventions at the moment :-/